lib/grakn.ex

```diff
@@ -48,6 +48,13 @@ defmodule Grakn do
   @spec query(conn(), Grakn.Query.t(), Keyword.t()) :: any()
   def query(conn, query, opts \\ []) do
     DBConnection.execute(get_conn(conn), query, [], with_transaction_config(opts))
+    |> case do
```

Comment:
Really Nit: 

What do you think about this style thing https://github.com/lexmag/elixir-style-guide#needless-pipeline ?

Reply:
This is actually a case for `with`:

```elixir
with {:ok, _query, result} <- 
      DBConnection.execute(get_conn(conn), query, [], with_transaction_config(opts)) do
  {:ok, result}
end
```